torch/_dynamo/variables/misc.py

```diff
@@ -798,6 +798,15 @@ def get_np_to_tnp_map():
     return np_fn_to_tnp_fn
 
 
+@functools.lru_cache(maxsize=1)
```

Comment:
any reason not to make this just a global variable?

Reply:
Just that the global variable would need to be defined after the `NumpyDTypeVariable` and `NumpyTypeInfoVariable` declarations below. I will move it to a global variable though we don't need to use this pattern here.